options.html

```
@@ -22,30 +33,41 @@ <h2>Farve Gradient</h2>
                     <div class="color-input-group">
                         <label for="color1">Første farve:</label>
                         <div class="color-picker-wrapper">
-                            <input type="color" id="color1" value="#48bb78">
-                            <input type="text" id="color1-hex" value="#48bb78" maxlength="7" placeholder="#000000">
+                            <input type="color" id="color1" value="#b5c99a">
+                            <input type="text" id="color1-hex" value="#b5c99a" maxlength="7" placeholder="#000000">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)